### DISCUSSION TITLE
Change the keybindings inside the session manager

### DISCUSSION THREAD

**Discussion by lucmos** [zellij-org/zellij#3211] [Q&A]
How can I change the keybindings inside the session manager?

I remapped Ctrl r to the global resize, thus now it overlaps with the session manager "rename session", for example.

![image](https://github.com/zellij-org/zellij/assets/11019190/6ad8a230-0849-402a-bf8c-bfcda566a542)

**Comment by imsnif** [comment]
The keybindings inside the session manager are unfortunately not configurable. The session-manager though is implemented as a plugin, so you can fork it and change it to behave however you like. Sorry that this is not as trivial as providing a config, but the burden of maintenance on this project is large and we have to choose where our work is concentrated on very carefully.

**Reply by lucmos** [reply]
I absolutely understand! Thank you for the fast and detailed answer :smiley:
